src/benchq/resource_estimation/graph/graph_estimator.py

```diff
@@ -23,12 +23,23 @@ def substrate_scheduler(graph: nx.Graph) -> TwoRowSubstrateScheduler:
     return scheduler_only_compiler
 
 
+@dataclass
+class GraphData:
+    """Contains minimal set of data to get a resource estimate for a graph."""
+
+    max_node_degree: int
```

Comment:
Actually, it was ok to talk about about "max graph degree", see:

The maximum degree of a graph  G denoted by \Delta(G), and the minimum degree of a graph, denoted by  \delta (G), are the maximum and minimum of its vertices' degrees.
([source](https://en.wikipedia.org/wiki/Degree_(graph_theory)))

And since we care about the properties of the graph as a whole, I think it would be better to stick to the old language.

Reply:
🙏